joplin/webpack.build.js

```diff
@@ -0,0 +1,38 @@
+var BundleTracker = require("webpack-bundle-tracker");
```

Comment:
Little bit of copypasta between this and webpack.dev.js, any ideas on how to reduce that are super welcome

Reply:
let's make a techdebt issue

Reply:
https://github.com/cityofaustin/techstack/issues/605